builder/docker/step_temp_dir.go

```diff
@@ -18,7 +19,23 @@ func (s *StepTempDir) Run(state multistep.StateBag) multistep.StepAction {
 	ui := state.Get("ui").(packer.Ui)
 
 	ui.Say("Creating a temporary directory for sharing data...")
-	td, err := ioutil.TempDir("", "packer-docker")
+
+	var err error
+	var td string
+	tempenv := "TMPDIR"
+	if runtime.GOOS == "windows" {
```

Comment:
Why do we need to change the environment variable based on OS? Could we not use something like `PACKER_TMP_DIR` on all OSes?